spec/helpers.rb

```diff
@@ -0,0 +1,5 @@
+module Helpers
+  def json
+    JSON.parse(response.body)
```

Comment:
This method should validate that response is not empty just in case, you can use either `if else` or you can use the safe navigator or null propagation operator like `response&.body`